ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/clock.schema.yml

```
@@ -0,0 +1,10 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  clock:
+    type: dict
+    keys:
+      timezone:
+        type: str  
```

Add trailing whitespace
```suggestion
        type: str  

```